### ISSUE TITLE
Building in Mac OSX

### ISSUE THREAD

**Issue by adammendoza** [vatioz/AudibleBookmarks#4]
I cloned the repo and I tried to build it but I'm getting several errors. Do I need to make changes to build on Mac?

**Comment by vatioz** [comment]
Hi, I tried to create the project crossplatform, but later realized, that it is dependent on Audible official Windows 10 app (it uses its database file). I kind of thought there is some similar app on Mac but apparently there isn't. Unfortunately I don't have access to Mac so I don't know for sure how audible works on Mac.

So I am afraid this project currently is not suited for Mac development. Let me know if you know about anything that would change what I know about audible on Mac.

**Comment by adammendoza** [comment]
I think on Mac it uses iTunes (I just use the iOS app). I was able to get the app going but can't do anything without the database.

**Comment by vatioz** [comment]
Yes, that's exactly the issue - it's all based on the database and unless we can find some access to it, this tool can't do much.
AFAIK, iTunes gives you access to the books but not to the bookmarks.

It's good to hear you've been able to run it, though ;)

**Comment by mahajangaurav** [comment]
Amazing solution and thanks for putting it together. I am running into a an issue though. I feel the issue I am facing is unique and might not have a solution but thought will try asking nonetheless. The problem is that my account is on audible.in. When I sign in to the windows app it does not show audible.in as an option in the list of marketplaces at sign-in. Signing in to audible.com does not bring my audible.in books. I presently listen to audible via my iphone and a samsung tablet (and did sign in to those using he audible.in marketplace). Any chance we could get to the SQL database on either of these devices?
Thanks in advance!
